packages/benchmarks/src/quizQuestions/bin/reportMongoDbUniversityBadgeQuestionBenchmark.ts

```diff
@@ -0,0 +1,230 @@
+import { BRAINTRUST_ENV_VARS } from "../../envVars";
+import { assertEnvVars } from "mongodb-rag-core";
+import {
+  ExperimentResult,
+  getBraintrustExperimentResults,
+} from "../../reporting/getBraintrustExperimentResults";
+import path from "path";
+import fs from "fs";
+import { createObjectCsvWriter } from "csv-writer";
+import { aggregateScoresByTag } from "../../reporting/aggregateScoresByTag";
+import { convertTagStatsToFlatObject } from "../../reporting/convertTagStatsToFlatObject";
+import {
+  QuizQuestionEvalCase,
+  QuizQuestionTaskOutput,
+} from "../QuizQuestionEval";
+import { aggregateExperimentScoreMean } from "../../reporting/aggregateExperimentScoreMean";
+
+const { BRAINTRUST_API_KEY } = assertEnvVars(BRAINTRUST_ENV_VARS);
+const projectName = "mongodb-multiple-choice";
+const experiments = [
+  {
+    model: "GPT-4o",
+    experimentName: "gpt-4o-badge-631d3a9b",
+  },
+  {
+    model: "Claude 3.5 Sonnet v2",
+    experimentName: "claude-35-sonnet-v2-badge-cb743d9f",
+  },
+  {
+    model: "Claude 3.5 Sonnet",
+    experimentName: "claude-35-sonnet-badge-f3427e16",
+  },
+  { model: "Gemini 2 Flash", experimentName: "gemini-2-flash-badge-76fea4f5" },
+  {
+    model: "Claude 3.5 Haiku",
+    experimentName: "claude-35-haiku-badge-4f4d32bb",
+  },
+  { model: "Nova Pro v1", experimentName: "nova-pro-v1:0-badge-e76a0833" },
+  { model: "Llama 3.1 70B", experimentName: "llama-3.1-70b-badge-f2e28e86" },
+  { model: "Llama 3.2 90B", experimentName: "llama-3.2-90b-badge-81111f12" },
+  {
+    model: "Claude 3.5 Haiku",
+    experimentName: "claude-35-haiku-badge-4f4d32bb",
+  },
+  {
+    model: "Gemini 1.5 Flash",
+    experimentName: "gemini-1.5-flash-002-badge-e0141bec",
+  },
+  { model: "Nova Lite v1", experimentName: "nova-lite-v1:0-badge-c896c5f3" },
+  { model: "Llama 3 70B", experimentName: "llama-3-70b-badge-54545f72" },
+  { model: "GPT-4o Mini", experimentName: "gpt-4o-mini-badge" },
+  {
+    model: "Gemini 1.5 Pro",
+    experimentName: "gemini-1.5-pro-002-badge-fc8268f2",
+  },
+  {
+    model: "GPT-35 Turbo 16k",
+    experimentName: "gpt-35-turbo-16k-badge-6282561d",
+  },
+  {
+    model: "Gemini 1.0 Pro",
+    experimentName: "gemini-1.0-pro-002-badge-62c646b1",
+  },
+  { model: "Nova Micro v1", experimentName: "nova-micro-v1:0-badge-e415a9d7" },
+  {
+    model: "Mistral Large 2",
+    experimentName: "mistral-large-2-badge-2526d48b",
+  },
+  {
+    model: "Claude 3 Sonnet",
+    experimentName: "claude-3-sonnet-badge-a8791d43",
+  },
+];
+
+const basePathOut = path.resolve(__dirname, "..", "..", "..", "testData");
+
+/**
+Ensures the output directory exists.
+
+@param dirPath The path to the directory to ensure.
+*/
+function ensureOutputDirectory(dirPath: string): void {
+  if (!fs.existsSync(dirPath)) {
+    fs.mkdirSync(dirPath, { recursive: true });
+  }
+}
```

Comment:
Seems like lots of duplicated functions/types between these two report files. Any plans to consolidate?